au/BUILD.bazel

```diff
@@ -19,7 +19,8 @@ load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 
 cc_library(
     name = "au",
-    hdrs = ["au.hh"],
+    hdrs = ["code/au/au.hh"],
+    includes = ["code"],
```

Comment:
I thought about creating a bazel macro to handle this repetition, but I ended up deciding that direct representation might be easier to understand and maintain.